ocaml/xapi/updateinfo.mli

```diff
@@ -113,19 +117,30 @@ module Severity : sig
   val of_string : string -> t
 end
 
-(** The metadata of one update in updateinfo *)
+(** The type of [guidance] in updateinfo metadata. *)
+module GuidanceInUpdateInfo : sig
```

Comment:
add an "s" -> `GuidancesInUpdateInfo` ?

Reply:
I think using `guidance` is consistent with the name of the field.